src/data/rooms.json

```diff
@@ -4,11 +4,17 @@
     "title": "Webex Component User",
     "roomType": "group"
   },
-  "default-activities": [],
-  "direct": {
-    "ID": "direct",
+  "default-activities": ["default", "no-header", "long", "multi-line"],
+  "empty-space": {
+    "ID": "empty-space",
+    "title": "Webex Component User",
```

Comment:
The title for this space can be something else like `Empty Test Room`. To differentiate between group spaces and 1:1 spaces